tools/generate_bigdump.php

```diff
@@ -43,8 +43,23 @@
    exit();
 }
 
+if (in_array('--help', $_SERVER['argv'])) {
+   die("usage: ".$_SERVER['argv'][0]."  [ --user=glpi ] [ --pass=glpi ]\n");
+}
+
+if ($_SERVER['argc']>1) {
+   for ($i=1; $i<count($_SERVER['argv']); $i++) {
+      $it           = explode("=", $argv[$i], 2);
+      $it[0]        = preg_replace('/^--/', '', $it[0]);
+      $args[$it[0]] = (isset($it[1]) ? $it[1] : true);
```

Comment:
I would have put `''` as default value, or `null`. If using `null`, then the `array_key_exists` function have to be used instead of `isset` on lines 58/59.

Reply:
Since this is a copy/paste from another existing script (cliupdate I guess), I'd prefer to keep as it.
I have in mind to include a lib that handles all that for us in the future (I did not find time to work on that yet).